manifests/init.pp

```diff
@@ -1,20 +1,49 @@
+# == Class: homebrew
+#
+# This class is able to install and configure homebrew and brew cask on node.
```

Comment:
Remove " on node." since I had to double-take to make sure there was no new nodejs dependency :P